TUTORIAL.md

```diff
@@ -0,0 +1,396 @@
+# CesiumJS and Webpack
+
+[Webpack](https://webpack.js.org/) is a popular and powerful tool for bundling JavaScript modules. It allows developers to structure their code and assets in an intuitive way and to load different kinds of files as needed with simple require statements. When building, it will trace code dependencies and pack these modules into one or more bundles that are loaded by the web browser.
+
+In this tutorial, we will build a simple web app from the ground up using webpack, and then cover the steps to integrate the [Cesium npm module](https://www.npmjs.com/package/cesium). This is a good place to start if you’d like to use CesiumJS to develop a web application. If you’re new to Cesium and are looking to learn to build your first sample app, take a look at our [Getting Started Tutorial](https://cesium.com/learn/cesiumjs-learn/cesiumjs-quickstart/).
```

Comment:
`Cesium npm module` -> `Cesium NPM module`

Reply:
Also please be consistent in the use of Cesium vs CesiumJS. I would recommend using CesiumJS throughout.

Reply:
I also took some time to look over the npm documentation. I believe that @thw0rted is correct here. "Cesium npm package" is technically the right phrase to use. @shehzan10 is there other npm documentation that you can share with me that suggests otherwise? For now, I added a commit with the phrasing "Cesium npm package". I can always update this later if necessary.